Makefile

```diff
@@ -235,6 +239,10 @@ $(SETUP_ENVTEST): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR); go get sigs.k8s.io/controller-runtime/tools/setup-envtest; \
 	go build -tags=tools -o $(BIN_DIR)/setup-envtest sigs.k8s.io/controller-runtime/tools/setup-envtest
 
+$(GINKGO): $(TOOLS_DIR)/go.mod
+	cd $(TOOLS_DIR); go get github.com/onsi/ginkgo/v2/ginkgo@v2.6.0
```

Comment:
`cd; cmd` pattern should always be written as `cd && cmd`, so if the cd fails it doesn't continue execution in wrong path.

Reply:
fixed